sway-fmt-v2/src/items/item_abi.rs

```diff
@@ -1,8 +1,9 @@
 use crate::fmt::{Format, FormattedCode, Formatter};
 use sway_parse::ItemAbi;
+use sway_types::Spanned;
 
 impl Format for ItemAbi {
     fn format(&self, _formatter: &mut Formatter) -> FormattedCode {
-        todo!()
+        self.span().as_str().to_string()
```

Comment:
Is this change relevant to this PR, (if not) maybe we can leave them as `todo!()`.

Reply:
Ah, these are so that the test included will pass. I can remove them and the test for the time being since the test won't pass the way we want to until both `abi` and this PR land